Add styling for tab in tab views (#5971)
sulu/sulu: UPGRADE.md

## 2.3

### React Tabs skin was removed

> Previously the `tabs` had a `skin` property. This property was replaced with the `type` property.
> Currently there are three `types` available: `root`, `nested` and `inline`. If you have used one
> of the skins, be sure to replace them with the corresponding type. 

Reviewer: ```suggestion
The `skin` prop and `small` prop of the `Tabs` component was replaced with a `type` prop.
Currently there are three `types` available: `root`, `nested` and `inline`. If you have used 
the `Tabs` component with a skin, be sure to replace them with the corresponding type. 
```

> `root` is the same as `skin='default'`.
> 
> `nested` is a new type, which is used for `nested` tabs.
> 
> `inline` is similar to `Tabs` with property `small=true` and `skin='transparent'`.

Reviewer: ```suggestion
- `type='root'` is the same as `skin='default'`.
- `type='inline'` is similar to `small=true` and `skin='transparent'`.
```
